string[]              joint_names
float64[]             start_joints
float64[]             end_joints
geometry_msgs/Pose    start_pose
geometry_msgs/Pose    end_pose
string                type
float64               max_velocity

---

string[] joint_names
trajectory_msgs/JointTrajectoryPoint[] points
int32 error_code

